packages/spor-linjetag-react-native/src/styling.ts

```diff
@@ -0,0 +1,121 @@
+import {
+  AltTransportFill18Icon,
+  AltTransportFill24Icon,
+  AltTransportFill30Icon,
+  BusFill18Icon,
+  BusFill24Icon,
+  BusFill30Icon,
+  ExpressBusFill18Icon,
+  ExpressBusFill24Icon,
+  ExpressBusFill30Icon,
+  FerryFill18Icon,
+  FerryFill24Icon,
+  FerryFill30Icon,
+  IconComponent,
+  SubwayFill18Icon,
+  SubwayFill24Icon,
+  SubwayFill30Icon,
+  TrainFill18Icon,
+  TrainFill24Icon,
+  TrainFill30Icon,
+  TramFill18Icon,
+  TramFill24Icon,
+  TramFill30Icon,
+  WalkFill18Icon,
+  WalkFill24Icon,
+  WalkFill30Icon,
+} from "@vygruppen/spor-icon-react-native";
+import { Theme } from "@vygruppen/spor-theme-react-native";
+import { Size, Variant } from "./types";
+
+type VariantStyling = {
+  icons: Record<Size, IconComponent>;
+  mainBackgroundColor: keyof Theme["colors"];
+  accentBackgroundColor: keyof Theme["colors"];
+  iconColor: "darkGrey" | "white";
+  borderColor?: keyof Theme["colors"];
+};
+
+export const variantStyling: Record<Variant, VariantStyling> = {
```

Comment:
Dette er strukturen jeg har satt opp for å dele designet på tvers av komponentene. Litt kjipt å ikke ha det samlokalisert, men fortsatt bedre enn å ha det i en egen `theme`-pakke. Her har vi i tillegg ikoner som må spesifiseres, som uansett ikke skal leve i en `theme`-pakke – så da kan vi likegodt bare ha alt her.

Reply:
Dette blir mye bedre 🙌